qiskit/circuit/parameter.py

```diff
@@ -25,10 +25,13 @@
 
 
 class Parameter(ParameterExpression):
-    """Parameter Class for variable parameters.
+    """A compile-time symbolic parameter.
```

Comment:
```suggestion
    """A symbolic parameter whose value must be determined at compile-time, i.e. before sending the associated quantum circuit to a quantum processing unit.
```

Reply:
Done (slightly differently) in 2c1cc17232a.